SKILL.tsv

```diff
@@ -921,114 +921,114 @@ SKILL_20150317_000920	Thrust
 SKILL_20150317_000938	Swash Buckling
-SKILL_20150317_000939	Hit shield to threathen enemy.
+SKILL_20150317_000939	Hit your shield to provoke the enemies surrounding you.
 SKILL_20150317_000940	Target #{SkillSR}#{nl}Max. provoke +#{CaptionRatio}# for #{CaptionTime}# seconds
-SKILL_20150317_000941	Reduce attack and increase defense.
+SKILL_20150317_000941	Decreases your attack, but increases your defense.
 SKILL_20150317_000942	Physical Defense + #{CaptionRatio}#{nl}Physical Damage - #{CaptionRatio2}# {nl}Duration #{CaptionTime}# seconds
 SKILL_20150317_000943	Shield Lob
 SKILL_20150317_000944	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl} Throw shield to attack enemy.
 SKILL_20150317_000945	High Guard
 SKILL_20150317_000946	Butterfly
 SKILL_20150317_000947	Wagon Wheel
-SKILL_20150317_000948	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Uppercut with weapon to Blow away enemy.
+SKILL_20150317_000948	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl} Uppercut with weapon to Blow away enemy.
 SKILL_20150317_000949	Cartar Stroke
-SKILL_20150317_000950	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl} Gather strength and hit the ground to attack enemy
+SKILL_20150317_000950	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl} Gather up power and use your weapon to attack the target by smashing it to the ground.
 SKILL_20150317_000951	Attack #{SkillAtkAdd}#{nl}Cast Time 0.5 seconds{nl}Splash #{SkillSR}#
 SKILL_20150317_000952	Crown
-SKILL_20150317_000953	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Smash enemy's head. Temporarily decreases the enemy's INT and SPR.
+SKILL_20150317_000953	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl} Release a powerful attack by smashing on the enemy's head. Temporarily decreases the enemy's INT and SPR.
 SKILL_20150317_000954	Attack #{SkillAtkAdd}#{nl}Splash #{SkillSR}#{nl}{#339999}{ol}[Shock]{/}{/} Duration #{CaptionTime}# seconds
 SKILL_20150317_000955	Cross Guard
-SKILL_20150317_000956	Take defensive stance using weapon.
+SKILL_20150317_000956	Use your weapon to take a defensive stance.
 SKILL_20150317_000957	Defense + #{CaptionRatio}#{nl}Block + #{CaptionRatio2}#
 SKILL_20150317_000958	Moulinet
-SKILL_20150317_000959	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl}Attack enemy continuously 5 times
+SKILL_20150317_000959	{#DD5500}{ol}[Physical] - [Slash]{/}{/}{nl} Deal five consecutive attacks in front of the enemy.
 SKILL_20150317_000960	Stabbing
-SKILL_20150317_000961	{#DD5500}{ol}[Physical] - [Stab]{/}{/}{nl}Attack enemy continuously using spear.
+SKILL_20150317_000961	{#DD5500}{ol}[Physical] - [Stab]{/}{/}{nl} Attack an enemy consecutively with a spear.
 SKILL_20150317_000962	Pierce
-SKILL_20150317_000963	{#DD5500}{ol}[Physical] - [Stab]{/}{/}{nl}Strongly stab enemy. {nl}Continuous attack bonus applies depending on size of enemy.
+SKILL_20150317_000963	{#DD5500}{ol}[Physical] - [Stab]{/}{/}{nl} Inflict damage on an enemy with a strong stab. {nl} Depending on the enemy's size, a bonus of continuous hits will apply.
```

Comment:
Inflicts?
Also,  "Additional attacks will/may be performed depending on the enemy's size"